src/python/dart/web/api/subscription.py

```diff
@@ -178,3 +181,20 @@ def subscription_service():
 def subscription_element_service():
     """ :rtype: dart.service.subscription.SubscriptionElementService """
     return current_app.dart_context.get(SubscriptionElementService)
+
+
+def create_nudge_subscription(subscription, dataset):
+    """ :type subscription: dart.model.subscription.Subscription
+        :type dataset: dart.model.dataset.Dataset
+        :rtype str
+    """
+    host_url = current_app.dart_context.config.get('nudge').get('url')
+    path = subscription.data.s3_path_start_prefix_inclusive if subscription.data.s3_path_start_prefix_inclusive else dataset.location
+    json_body = {
+        'Bucket': get_bucket_name(path),
+        'Prefix': get_key_name(path),
+        'Regex': subscription.data.s3_path_regex_filter,
```

Comment:
So dart does this thing where it allows both a prefix and a full regex that redefines the prefix. In nudge I made the regex match the remainder of the key after the prefix is matched.